### PR TITLE
[OUR415-286] Adds back print styling, extracts out pagination css (again 😅), and fixes search input redirect to /search page

### PR DESCRIPTION
**Pull request by rosschapman**
🧑‍🌾

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -40,5 +40,6 @@ module.exports = {
       { namedComponents: "arrow-function" },
     ],
     "@typescript-eslint/ban-ts-comment": "warn",
+    "@typescript-eslint/prefer-nullish-coalescing": "warn"
```
Comment (rosschapman) [inline .eslintrc.js:43]: Sneaking this in. Something @adriencyberspace and I discussed liked having.

**Review by adriencyberspace — CHANGES_REQUESTED** [review]
Can we address the styling issues caused by the search bar being in the navbar in this PR?

1. At smaller desktop screens, user is no longer able to access all nav links (search bar should take up less horizontal space and probably be flex width)
image
2. There should be 32px gap between Our415 logo and search bar
3. IMO there should be a 120px container around the logo so that the searchbar is always in the same place on the screen (there is something like this around the google translate widget already)
4. nit: searchbar should be same height as logo and with a white background (design changed)

File: .eslintrc.js [hunk 2]
```diff
@@ -40,5 +40,6 @@ module.exports = {
       { namedComponents: "arrow-function" },
     ],
     "@typescript-eslint/ban-ts-comment": "warn",
+    "@typescript-eslint/prefer-nullish-coalescing": "warn",
```
Comment (rosschapman) [inline .eslintrc.js:43]:
```suggestion
```

**Comment by rosschapman** [comment]
> Can we address the styling issues caused by the search bar being in the navbar in this PR?

@adriencyberspace Yep, definitely some style issues as things have moved around. I'm going to focus on this separately. Kyle has some new recommendations and spec for the navbar in https://www.notion.so/exygy/Update-mobile-menu-e7a2f2d9d23748baabd943e1f0b2f045?pvs=4.

**Pull request merged**